website/settings/defaults.py

```diff
@@ -1958,3 +1959,18 @@ class CeleryConfig:
 DS_METRICS_BASE_FOLDER = None
 REG_METRICS_OSF_TOKEN = None
 REG_METRICS_BASE_FOLDER = None
+
+STORAGE_WARNING_THRESHOLD = .9  # percent of maximum storage used before users get a warning message
```

Comment:
Could you add settings for STORAGE_LIMIT_PUBLIC and STORAGE_LIMIT_PRIVATE here as well? We had them in Corbin's PR, but I've asked that they get removed from there so we could add them here.